app/scripts/filters/strings.js

```diff
@@ -56,4 +56,14 @@ angular.module('openshiftConsole')
       }
       return str.indexOf('\n') !== -1;
     };
+  })
+  .filter('encodeBase64', function() {
+    return function(str) {
+      return window.btoa(str);
+    };
+  })
+  .filter('decodeBase64', function() {
+    return function(str) {
+      return window.btoa(str);
+    };
```

Comment:
Since we're only using these in the controllers, no need to wrap `window.btoa` in a filter. Just call it directly.